apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -179,6 +148,48 @@ function showHomepage() {
   );
 }
 
+/**
+ * Return the teacher announcement that we should pass into TeacherHomepage.
+ * @param {object} override - An optional override announcement.
+ * @return {object} An announcement to display.
+ */
+function getTeacherAnnouncement(override) {
```

Comment:
I'm sure there's a better way to scope this function.  @islemaster or @joshlory, thoughts?

Reply:
I'm not sure what you mean here.  Since this goes through our Webpack pipeline, this function is currently private to this file, which seems appropriately scoped to me.

Reply:
Oh, that's good to know.  Thanks.